core/common/src/main/java/alluxio/exception/status/AlluxioStatusException.java

```diff
@@ -84,6 +85,86 @@ public ExceptionStatus getStatus() {
   }
 
   /**
+   * @return the Thrift representation of this exception
+   */
+  public AlluxioTException toThrift() {
+    return new AlluxioTException(getMessage(), ExceptionStatus.toThrift(mStatus));
+  }
+
+  /**
+   * @return a specific {@link AlluxioException} corresponding to this exception if there is one;
+   *         otherwise return a generic {@link AlluxioException}
+   */
+  public AlluxioException toAlluxioException() {
+    switch (mStatus) {
+      // Fall throughs are intentional.
+      case PERMISSION_DENIED:
+      case UNAUTHENTICATED:
+        return new AccessControlException(getMessage());
```

Comment:
They are the only `AlluxioStatusExceptions` which should always be translated to specific `AlluxioExceptions`.